types/wordpress__components/dropdown-menu/index.d.ts

```diff
@@ -33,6 +35,37 @@ declare namespace DropdownMenu {
          * @defaultValue "top center"
          */
         position?: Popover.Position | undefined;
+
+        /**
+         * Object to pass as props to the underlying NavigableMenu component.
+         */
+        menuProps?: Partial<NavigableMenu.Props> | undefined;
+
+        /**
+         * Object to pass as props to the underlying Popover component.
+         */
+        popoverProps?: Partial<Popover.Props> | undefined;
+
+        /**
+         * Object to pass as props to the underlying toggle Button component.
+         */
+        toggleProps?: Partial<Button.ButtonProps> | undefined;
+
+        /**
+         * Whether to disable the arrow down key to open the dropdown menu.
+         * @defaultValue false
+         */
+        disableOpenOnArrowDown?: boolean | undefined;
+
+        /**
+         * The text to pass as prop to the underlying toggle Button component.
+         */
+        text?: string | undefined;
```

Comment:
I don't see this mentioned in the [component README](noIcons).

Reply:
Removed the undocumented props